manager/manager.go

```diff
@@ -171,7 +173,17 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 	if err != nil {
 		glog.Warningf("Unable to connect to Docker: %v", err)
 	}
-	context := fs.Context{DockerRoot: docker.RootDir(), DockerInfo: dockerInfo}
+
+	// TODO: handle case where we're not using device mapper as the storage driver
```

Comment:
The docker driver should perform this initialization ideally. The rest of cadvisor should not depend on docker semantics (ideally)